File: 프로그래머스/src/프로그래머스_02252022/없는숫자더하기.cpp
#include <string>
#include <vector>

using namespace std;

int solution(vector<int> numbers)
{
    int answer = 0;
    vector<int> counter(10, 0); // 0~9까지 cnt값을 0으로 초기화

    for (auto n : numbers)
        counter[n] += 1;
    for (int i = 0; i < 10; i++)
        answer += (counter[i] == 0) ? i : 0;

    return answer;
}

File: 2026/Level1/없는숫자더하기.py
# https://school.programmers.co.kr/learn/courses/30/lessons/86051?language=python3
def solution(numbers):
    answer = 0
    exists = [False] * 10
    for x in numbers :
        exists[x] = True
    
    # print(exists)
    for x in range(1,10):
        if exists[x] == False:
            answer += x
    return answer